05-Data_Ethics.Rmd

```diff
@@ -159,39 +159,76 @@ Possible guidelines could include (based on @garrison_genomic_2019, @garrison_ge
+
+Patients need to make their own decisions about their health. They should be equipped with all information about their health but then respected and supported in their decision.
+This is otherwise includes informed consent [@commissioner_guide_2020], which means before a patient can truly consent they need to be fully informed of the risks, ins and outs of any procedure, treatment, or research study participation.
+
+## Code of Medical Ethics (1957) - Legal Binding of Patient-Physician Confidentiality
+
+1996 Health Insurance Portability and Accountability Act (HIPAA) protected health information and identifiable health information must not be shared with anyone outside of certain covered entities without consent. Covered entities include: clinicians, insurance companies, and health care government agencies.
 
-and this amazing timeline:
-https://www.usuhs.edu/sites/default/files/2021-06/TimelineofLandmarkEventsinBioethics.pdf
+## Genetic Information Nondiscrimination Act of 2008 (GINA)
 
-- informed consent etc. [@commissioner_guide_2020]
+The Genetic Information Nondiscrimination Act prohibits employers and health insurance companies from using genetic information to discriminate against individuals.
+This data would be highly dangerous and valuable if in the wrong hands. Health insurance companies are prohibited from obtaining genetic test data and using it to inform their premiums or other healthcare coverage related decisions.
 
-Code of Medical Ethics (1957) - Legal Binding of Patient-Physician Confidentiality 
+> defines “genetic test” to mean the “analysis of human DNA, RNA, chromosomes, proteins, or metabolites that detects genotypes, mutations, or chromosomal changes.” Genetic tests are used to detect gene variants associated with a specific disease or condition. For example, tests to determine whether an individual carries the genetic variant evidencing a predisposition to breast cancer—whether the individual has the BRCA1 or BRCA2 variant—or to determine whether an individual has a genetic variant associated with hereditary nonpolyposis colorectal cancer are genetic tests.
 
-1996 Health Insurance Portability and Accountability Act (HIPAA)
- - protected health information and identifiable health information must not be shared with anyone outside of certain covered entities without consent. Covered entities include: clinicians, insurance companies, and health care government agencies. 
- 
- Genetic Information Nondiscrimination Act of 2008 (GINA)
+From [@regulations_2010]
 
-2009 Health Information Technology for Economic and Clinical  Health (HITECH) Act
+This means as researchers or clinicians we need to keep these data secure and hidden. Data types protected under GINA include but are not limited to:
+
+- DNA sequencing or genotyping
+- RNA sequencing that shows genetic variation
+- Protein sequencing
+- Blood collection
+- Any type of biological material that would indicate heredity or any genetic variant
+
```

Comment:
This is great! I actually planned to put this in the data privacy chapter - where we talk about genetic data.  what do you think about that?

Reply:
Move this wherever you have envisioned it should be! No worries!

Reply:
OK cool! thanks